react/store/createReturnRequest/components/ItemsDetails.tsx

```diff
@@ -130,18 +146,27 @@ export const ItemsDetails = (itemToReturn: ItemToReturn) => {
           otherReason={currentItem?.returnReason?.otherReason ?? ''}
           onReasonChange={handleReasonChange}
         />
-        {/* TODO Intl */}
-        {reasonError && reasonErrorEmptyValue ? 'Reason is required' : null}
+        {reasonError && reasonErrorEmptyValue ? (
+          <CustomMessage
+            status="error"
+            message="store/return-app.return-item-details.dropdown-reason.error"
+          />
+        ) : null}
         {/* TODO user input when other & error */}
```

Comment:
We can delete this TODO. Your implementation for the validation in the other field covers that!